analytics/terraform/spark-k8s-operator/examples/karpenter/spark-mountpoint-s3/daemonset-with-configmap.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: s3-mount-script
+  namespace: spark-team-a
```

Comment:
We need to explain that users need to create dedicated namespace, service account, IAM role required to access the required S3 bucket for this deployment